nilearn/surface/tests/test_surface.py

```diff
@@ -104,7 +105,7 @@ def test_load_surf_data_file_nii_gii(tmp_path):
     gii = gifti.GiftiImage(darrays=[darray])
     gii.to_filename(filename_gii)
     assert_array_equal(load_surf_data(filename_gii), np.zeros((20,)))
-    os.remove(filename_gii)
+    Path.unlink(filename_gii)
 
     # test loading of data from empty gifti file
     fd_empty, filename_gii_empty = tempfile.mkstemp(
```

Comment:
Note to myself (not for this PR): the use of `tempfile` could be refactored with the proper pytest fixtures.

Would also avoid the manual removing of files for teardown.